open-api/rest-catalog-open-api.yaml

```diff
@@ -921,7 +921,8 @@ components:
           type: object
           description: Configured string to string map of properties for the namespace
           example: { "owner": "Hank Bendickson" }
-          default: { }
```

Comment:
nit: why are we removing `default` here and below?

Reply:
oops - re-added